docs/images_manjaro.md

```diff
@@ -1,4 +1,5 @@
 # ovos-image-arch-recipe
+**NOTE** This repo has been archived and will no longer be updated
```

Comment:
remove from user docs to avoid confusion? if someone finds the readme it says there its archived

Reply:
That was my first choice, but left it until someone said just what you said